.github/workflows/buildRelease.yml

```diff
@@ -51,6 +56,11 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.WORKFLOW_INVOCATION_TOKEN }}
       with:
         release_id: ${{ github.event.inputs.release_id }}
+    - name: Publish to PyPI
+      if: github.event.inputs.release_id != ''
+      uses: pypa/gh-action-pypi-publish@v1.5.1
+      with:
+        password: ${{ secrets.PYPI_API_TOKEN }}
```

Comment:
Do we have to specify the .whl package ? We don't want to push the torch package ? Just the torch-mlir one.

Reply:
The only other argument I see is `packages_dir`.  I can try that, though I'm not sure yet what the directory structure looks like on disk and whether these packages will just be in the same directory.  On a side note, the iteration time on this is _really_ slow, so this might take a while.